### PR TITLE
try fix undeffed isabstract on certain julia versions

### PR DESCRIPTION
**Pull request by KristofferC**
Ref https://github.com/JuliaLang/julia/commit/e2cceb619d100d57ccdd06aa1fdc6b4ec6d650a5#commitcomment-20813338 cc @tkelman

### CODE REVIEW

File: src/Compat.jl [hunk 1]
```diff
@@ -1797,7 +1797,7 @@ end
 
 module TypeUtils
     using ..Compat: @static
-    @static if isdefined(Core, :UnionAll)
+    @static if isdefined(Core, :isabstract)
```
Comment (tkelman) [inline src/Compat.jl]: `isabstract` isn't in Core

**Comment by KristofferC** [comment]
Right you are

File: src/Compat.jl (hunk 1 above)
Comment (KristofferC) [inline src/Compat.jl]: Right you are, thanks

**Comment by tkelman** [comment]
(probably should have squashed)

**Comment by KristofferC** [comment]
isch, indeed.. im so just to squash being the default

**Pull request merged**